pod-provider/backend/services/migration/2-2-0.js

```diff
@@ -0,0 +1,106 @@
+const urlJoin = require('url-join');
+const { MigrationService } = require('@semapps/migration');
+const CONFIG = require('../../config/config');
+
+const MIGRATION_VERSION = '2.2.0';
+
+module.exports = {
+  name: 'migration-2-2-0',
+  mixins: [MigrationService],
+  settings: {
+    baseUrl: CONFIG.BASE_URL
+  },
+  actions: {
+    async migrate(ctx) {
+      const { username } = ctx.params;
+      const accounts = await ctx.call('auth.account.find', { query: username === '*' ? undefined : { username } });
+
+      for (const { webId, username, version, ...rest } of accounts) {
+        if (version === MIGRATION_VERSION) {
+          this.logger.info(`Pod of ${webId} is already on v${MIGRATION_VERSION}, skipping...`);
+        } else {
+          this.logger.info(`Migrating Pod of ${webId} to v${MIGRATION_VERSION}...`);
+
+          ctx.meta.dataset = username;
+          ctx.meta.webId = webId;
+          ctx.meta.skipObjectsWatcher = true; // We don't want to trigger an Update
+
+          try {
+            // Delete old capabilities containers.
+            const containerUri = urlJoin(webId, 'data/capabilities');
+            await ctx.call('ldp.container.delete', { containerUri, webId: 'system' });
+
+            await ctx.call('repair.createMissingContainers', { username });
+            await this.actions.generatePrivateTypeIndex({ webId }, { parentCtx: ctx });
+            await ctx.call('type-registrations.resetFromRegistry', { webId });
+            await ctx.call('webacl.resource.refreshContainersRights', { webId });
+            await this.actions.generateDataRegistry({ webId }, { parentCtx: ctx });
+            await this.actions.attachDataRegistrationToContainers({ webId }, { parentCtx: ctx });
+            await ctx.call('repair.upgradeAllApps', { username });
```

Comment:
These are 2.1.0 migrations. They should be removed (as well as the actions below).
The `repair.createMissingContainers` call can be kept as it will be used to create the VC containers for existing users.